requirements.txt

```diff
@@ -24,7 +24,7 @@ certifi
 # Core
 six
 cachetools
-prometheus_client
+prometheus_client==0.5.0
```

Comment:
There should not be API changes between 0.5.* versions

What about using a range?

```
prometheus_client==0.5.*
```